classes/class-paysoncheckout-for-woocommerce-callbacks.php

```diff
@@ -199,6 +199,10 @@ public function process_recurring_payment( $payment_id, $payson_order ) {
 
 		$order = $this->get_wc_order_by_payment_id( $payment_id );
 
+		if ( ! $order ) {
+			return false;
+		}
```

Comment:
I see we have the `$payment_id`. Could we perhaps log something similar to this?

https://github.com/krokedil/paysoncheckout-for-woocommerce/blob/develop/classes/class-paysoncheckout-for-woocommerce-callbacks.php#L72